frontend/src/pages/singleProduct/singleProduct.js

```diff
@@ -28,9 +41,23 @@ function SingleProduct() {
               endPoint: "Single product",
             }}
           ></PathBar>
+          <Alert
```

Comment:
Think about creating component `Notification`, you can provide info about severity, notificationMessage and className and show it whenever `showNotification` flag is true. And component will take care for own disappearing.